src/components/categories/weather/Weather.vue

```diff
@@ -44,20 +51,27 @@
             }
         },
         created() {
-            this.$emit('addMessage', {author: "user", text: this.$t('weather.user.choiceWeather'), style: "default"});
-            this.$emit('addMessage', {author: "bot", text: this.$t('weather.bot.introduction'), style: "default"});
+            this.sendMessage("user", this.$t('weather.user.choiceWeather'), "default").then(() => {
+                this.sendMessage("bot", this.$t('weather.bot.introduction'), "default").then(() => {
+                    this.showCityDropdown = true;
+                })
+            })
         },
         methods: {
+            sendMessage(author, text, style = 'default') {
+                return new Promise((resolve) => {
```

Comment:
Tutaj i wszędzie wysyłanie wiadomości będzie uproszczone, przez to jak wyglądać będzie metodka sendMessage z messages.js